code/game/machinery/computer/robot.dm

```diff
@@ -204,7 +204,7 @@
 		robot["master_ai"] = R.connected_ai ? R.connected_ai.name : "None"
 		robot["hackable"] = 0
 		//Antag synths should be able to hack themselves and see their hacked status.
-		if(operator && istype(operator, /mob/living/silicon) && (operator.mind.special_role && operator.mind.original == operator))
+		if(operator && istype(operator, /mob/living/silicon/robot) && (operator.mind.special_role && operator.mind.original == operator) && (operator == R))

```

Comment:
Yeah, I saw it last second, and that's why I didn't do isrobot.
I threw up the DNM because I noticed it after making the PR.